demos/select.html

```diff
@@ -134,32 +134,6 @@ <h2 class="mdc-typography--title">Fully-Featured Component</h2>
           <label for="disabled">Disabled</label>
         </div>
       </section>
-
-      <section class="example">
-        <h2 class="mdc-typography--title">CSS Only</h2>
-        <section class="box-demo-wrapper">
-          <div id="demo-css-only-select" class="mdc-select">
-            <select class="mdc-select__surface">
-              <option value="" selected>Pick a food group</option>
-              <option value="grains">Bread, Cereal, Rice, and Pasta</option>
-              <option value="vegetables" disabled>Vegetables</option>
-              <option value="fruit">Fruit</option>
-              <option value="dairy">Milk, Yogurt, and Cheese</option>
-              <option value="meat">Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts</option>
-              <option value="fats">Fats, Oils, and Sweets</option>
-            </select>
-            <div class="mdc-select__bottom-line"></div>
-          </div>
-        </section>
-        <div>
-          <input type="checkbox" id="css-alternate-colors">
```

Comment:
You should also remove the JS that references these checkboxes that are no longer present.